extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -288,7 +289,12 @@ public function getArticle(){
 			if ( !empty( $relatedPages ) ) {
 				$data[ 'relatedPages' ] = $relatedPages;
 			}
-		} catch (WikiaHttpException $exception) {
+
+			if ( $title->isMainPage() ) {
```

Comment:
We would pick the communities via optimizely which works only on client side. We can add a wg variable but we would need to change it's value in WF manually. So we would need to sync it in two places. We can measure if it's slowing down mercury a lot but my guess would be it won't because we have caching enabled.